src/styles/components/DataGridWithInfinteScroll.scss

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+@import "../common/colors";
+@import "../common/fonts";
+@import "../common/borders";
+
+$csg-row-height: 48px;
```

Comment:
Why do we need these overrides, can't we use existing variables available.  
e.g. what is long form of `csg` ?

Reply:
Removed this, and using the existing one